pkg/kubelet/logs/container_log_manager.go

```diff
@@ -176,19 +176,108 @@ func (c *containerLogManager) Start() {
 	}, logMonitorPeriod)
 }
 
+type containerWithLog struct {
+	ID        string
+	CreatedAt int64
+	logPath   string
+	State     runtimeapi.ContainerState
+}
+
+func (c *containerLogManager) addContainerToMap(container *runtimeapi.Container, path string, containersByName map[string][]containerWithLog) {
+	if container.Metadata != nil {
+		key := container.Metadata.Name + " " + container.PodSandboxId
```

Comment:
The separator should be a `_` so we can visually see the key.

Reply:
Comment addressed in the next update.